README.md

```diff
@@ -89,6 +89,19 @@ Thats it, You now have TIDoS at your service. Fire it up using:
 python2 tidos.py
 ```
 
+#### Docker image :
+
+You can pull dockerized TIDoS-Framework from dockerhub :
```

Comment:
References to dockerfile from the original repo is required, not the external one which isn't controlled by project author.

Reply:
Noted. switch to manual deployment instead of pulling from official repo